graphql/resolvers.go

```diff
@@ -934,6 +949,14 @@ func (r *taskResolver) PatchNumber(ctx context.Context, obj *restModel.APITask)
 	return &patch.PatchNumber, nil
 }
 
+func (r *taskResolver) FailedTestCount(ctx context.Context, obj *restModel.APITask) (int, error) {
+	failedTestCount, err := r.sc.GetTestCountByTaskIdAndFilters(*obj.Id, "", []string{evergreen.TestFailedStatus}, obj.Execution)
+	if err != nil {
+		return 0, InternalServerError.Send(ctx, fmt.Sprintf("Error getting tests for failedTestCount: %s", err.Error()))
+	}
+	return failedTestCount, nil
+}
+
```

Comment:
amazing. this way we can poll this resolver. this is exactly what i mean when i describe putting total task count in its own resolver.